qgis.desktop

```diff
@@ -5,7 +5,7 @@ Name=QGIS Desktop
 GenericName=Geographic Information System
 Icon=/opt/conda/share/qgis/images/icons/qgis-icon-512x512.png
 TryExec=qgis
-Exec=qgis %F
+Exec=/home/qgis.bash %F
```

Comment:
Execute from somewhere other than `/home`

Reply:
I'd suggest putting this under `/usr/local/bin`, but with a different name (`maximized-qgis`) or similar.

Reply:
/usr/local/bim seems good path, I have just updated it